hardhat.config.ts

```diff
@@ -70,7 +71,7 @@ const userConfig: HardhatUserConfig = {
         target: "ethers-v6",
     },
     solidity: {
-        compilers: [{ version: primarySolidityVersion, settings: soliditySettings }, { version: "0.6.12" }, { version: "0.5.17" }],
+        compilers: [{ version: primarySolidityVersion, settings: soliditySettings }, { version: defaultSolidityVersion }],
```

Comment:
Shouldn't this be:

```suggestion
        compilers: [{ version: solidityVersion, settings: soliditySettings }],
```

I don't think we make use of multiple compilers, right?

Reply:
We do us for some of the OZ files which won't compile with `v0.8.19`. If we don't use the mentioned `defaultSolidityVersion`, it will result in the below error:

```
The Solidity version pragma statement in these files doesn't match any of the configured compilers in your config. Change the pragma or configure additional compiler versions in your hardhat config.

  * contracts/test/ERC20Token.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/token/ERC20/ERC20.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/token/ERC20/IERC20.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/math/SafeMath.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/utils/Context.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/proxy/UpgradeableProxy.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/proxy/Proxy.sol (>=0.6.0 <0.8.0)
  * @openzeppelin/contracts/utils/Address.sol (>=0.6.2 <0.8.0)

These files and its dependencies cannot be compiled with your config. This can happen because they have incompatible Solidity pragmas, or don't match any of your configured Solidity compilers.

  * contracts/test/TestImports.sol
```